examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -364,12 +364,16 @@ function default_remaining_tendency!(Yₜ, Y, p, t)
     end
     @. ᶠω¹² += ᶠcurlᵥ(ᶜuₕ)
 
-    # TODO: Modify to account for topography
-    @. ᶠu¹² = Geometry.Contravariant12Vector(ᶠinterp(ᶜuₕ))
-    @. ᶠu³ = Geometry.Contravariant3Vector(ᶠw)
+    fuvw = @. Geometry.Covariant123Vector(ᶠinterp(ᶜuₕ)) +
+       Geometry.Covariant123Vector(ᶠw)
+    cuvw = @. Geometry.Covariant123Vector(ᶜuₕ) +
+       Geometry.Covariant123Vector(ᶜinterp(ᶠw))
+    @. ᶠu¹² = Geometry.project(Geometry.Contravariant12Axis(), fuvw)
+    @. ᶠu³ = Geometry.project(Geometry.Contravariant3Axis(), fuvw)
 
     @. Yₜ.c.uₕ -=
-        ᶜinterp(ᶠω¹² × ᶠu³) + (ᶜf + ᶜω³) × Geometry.Contravariant12Vector(ᶜuₕ)
+        ᶜinterp(ᶠω¹² × ᶠu³) +
+        (ᶜf + ᶜω³) × (Geometry.project(Geometry.Contravariant12Axis(), cuvw))
```

Comment:
```suggestion
        ᶜinterp(ᶠω¹² × ᶠu³) +
        (ᶜf + ᶜω³) × (Geometry.project(Geometry.Contravariant12Axis(), ᶜuvw))
```
is already computed above